### PR TITLE
Hook members

### PR DESCRIPTION
**Pull request by JosueRodriguezGarcia**

### CODE REVIEW

File: src/main/java/trello/ui/pages/LoggedOutPage.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * LoggedOutPage class.
+ *
+ * @author Josue Rodriguez.
+ * @version 0.0.1
+ */
+public class LoggedOutPage extends BasePage {
+
+    @FindBy(linkText = "Log In")
```
Comment (MelissaRoman06) [inline src/main/java/trello/ui/pages/LoggedOutPage.java]: Please change the type of locator.

Comment (JosueRodriguezGarcia) [reply src/main/java/trello/ui/pages/LoggedOutPage.java]: Solved

File: src/test/resources/features/card.feature [hunk 2]
```diff
@@ -1,15 +1,38 @@
...
      Then the checklist section is displayed on the card details
+
+  @create-board @create-list @create-card @delete-board
+  Scenario: Assign a date to a card
+    Given I log in as admin user
+     When I select BoardTest board
+      And I select CardTest card
+      And I assign a due date
+        | Date     | One day from now |
+        | Reminder | None              |
```
Comment (raulchoque-eng) [inline src/test/resources/features/card.feature]: Please, justify this "|" chard.

File: src/test/java/trello/steps/CardSteps.java [hunk 3]
```diff
@@ -33,7 +41,13 @@
     private BoardPage boardPage;
     private CardModal cardModal;
     private CheckListModal checkListModal;
+    private DueDataModal dueDataModal;
     private Context context;
+    private MemberModal memberModal;
+    private InviteToBoardModal inviteToBoardModal;
+
+
+
```
Comment (raulchoque-eng) [inline src/test/java/trello/steps/CardSteps.java]: Delete these white lines, please.

File: src/main/java/trello/ui/pages/modal/InviteToBoardModal.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+     *
+     * @return a BoardPage.
+     */
+    public BoardPage clickSendInvitationButton() {
+        sendInvitationButton.click();
+        return new BoardPage();
+    }
+
+    @Override
+    protected void waitUntilPageObjectIsLoaded() {
```
Comment (raulchoque-eng) [inline src/main/java/trello/ui/pages/modal/InviteToBoardModal.java:66]: Comment this method, please.

File: src/main/java/trello/ui/pages/BoardPage.java [hunk 5]
```diff
@@ -100,8 +103,13 @@
     @FindBy(className = "js-open-add-list")
     private WebElement addAnotherList;
 
+
```
Comment (raulchoque-eng) [inline src/main/java/trello/ui/pages/BoardPage.java]: Delete this line, please.

File: src/main/java/core/selenium/util/datecomponents/YearComponent.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+/**
+ * YearComponent class.
+ *
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1
+ */
+public class YearComponent implements IDateComponent {
+
+    @Override
+    public String newDate(final int num) {
```
Comment (raulchoque-eng) [inline src/main/java/core/selenium/util/datecomponents/YearComponent.java:34]: Comment this method, please.

File: src/main/java/core/selenium/util/datecomponents/MonthComponent.java [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+/**
+ * MonthComponent class.
+ *
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1
+ */
+public class MonthComponent implements IDateComponent {
+
+    @Override
+    public String newDate(final int num) {
```
Comment (raulchoque-eng) [inline src/main/java/core/selenium/util/datecomponents/MonthComponent.java:34]: Comment this method, please.

File: src/main/java/core/selenium/util/datecomponents/HourComponent.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+/**
+ * HourComponent class.
+ *
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1
+ */
+public class HourComponent implements IDateComponent {
+
+    @Override
+    public String newDate(final int num) {
```
Comment (raulchoque-eng) [inline src/main/java/core/selenium/util/datecomponents/HourComponent.java:34]: Comment this method, please.

File: src/main/java/core/selenium/util/datecomponents/DayComponent.java [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+/**
+ * DayComponent class.
+ *
+ * @author Josue Rodriguez Garcia.
+ * @version 0.0.1
+ */
+public class DayComponent implements IDateComponent {
+
+    @Override
+    public String newDate(final int num) {
```
Comment (raulchoque-eng) [inline src/main/java/core/selenium/util/datecomponents/DayComponent.java:34]: Comment this method, please.

File: src/test/resources/features/card.feature (hunk 2 above)
Comment (JosueRodriguezGarcia) [reply src/test/resources/features/card.feature]: Solved

File: src/test/java/trello/steps/CardSteps.java (hunk 3 above)
Comment (JosueRodriguezGarcia) [reply src/test/java/trello/steps/CardSteps.java]: Solved

File: src/main/java/trello/ui/pages/modal/InviteToBoardModal.java (hunk 4 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/trello/ui/pages/modal/InviteToBoardModal.java:66]: Solved

File: src/main/java/trello/ui/pages/BoardPage.java (hunk 5 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/trello/ui/pages/BoardPage.java]: Solved

File: src/main/java/core/selenium/util/datecomponents/YearComponent.java (hunk 6 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/core/selenium/util/datecomponents/YearComponent.java:34]: Solved

File: src/main/java/core/selenium/util/datecomponents/MonthComponent.java (hunk 7 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/core/selenium/util/datecomponents/MonthComponent.java:34]: Solved

File: src/main/java/core/selenium/util/datecomponents/HourComponent.java (hunk 8 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/core/selenium/util/datecomponents/HourComponent.java:34]: Solved

File: src/main/java/core/selenium/util/datecomponents/DayComponent.java (hunk 9 above)
Comment (JosueRodriguezGarcia) [reply src/main/java/core/selenium/util/datecomponents/DayComponent.java:34]: Solved

**Review by MelissaRoman06 — APPROVED** [review]

**Pull request merged**